exercises/practice/bowling/.meta/template.j2

```diff
@@ -5,8 +5,13 @@
         rolls = {{ input["previousRolls"] }}
         game = self.roll_new_game(rolls)
         {% if case is error_case -%}
+        {% set property = case["property"] -%}
         with self.assertRaisesWithMessage(Exception):
+            {% if property == 'score' -%}
+            game.score()
+            {% else -%}
```

Comment:
Would it make sense for this to explicitly check the property value?

Reply:
@IsaacG - The CI for this has not run (as previously asked -- please run the CI and make sure it passes before reviewing.  If you cannot run the CI, please do not review.).

Reply:
I do believe I cannot run the CI.

Reply:
Please refrain from reviewing PRs that require a manually triggered CI run then.  Thanks.